### ISSUE TITLE
Website Redesign

### ISSUE THREAD

**Issue by matronator** [PrismJS/prism#3577]
**Motivation**

The website in its current form is kinda ugly (especially the header is a mess that's difficult to navigate visually) and looks a bit outdated.

**Description**

Redesign the website to a cleaner, nicer and more modern style.

**Alternatives**

Not look at the website, ehh?

**Comment by FirdausJaimi** [comment]
Assign me if you want. I m ready to work on it

**Comment by s-beji** [comment]
I agree it would be wise to modernize the website appearance and improve the navigation and the docs since while i was using it at first i was rather confused where to find the resources i required.
